client/views/newEpisode.js

```diff
@@ -2,6 +2,12 @@ Template.newEpisode.events({
   "submit form": function(e, t) {
     e.preventDefault();
     data = SimpleForm.processForm(e.target);
+
+    if (Episodes.findOne({episodeId: data.episodeId})) {
```

Comment:
Honestly, you could probably just do an upsert here and ditch the entire if block.